sdk/core/Azure.Core.Experimental/tests/LowLevelClient/PetStoreClient.cs

```diff
@@ -143,6 +145,60 @@ public virtual Response GetPet(string id, RequestOptions options = null)
             }
         }
 
+#pragma warning disable AZC0002
+        public virtual Operation<BinaryData> ImportPet(RequestContent content, RequestOptions options = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("PetStoreClient.ImportPet");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateImportPetRequest(content);
+                return LowLevelOperationHelpers.ProcessMessage(Pipeline, message, _clientDiagnostics, "PetStoreClient.ImportPet", OperationFinalStateVia.AzureAsyncOperation, options);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+#pragma warning disable AZC0002
+        public virtual async Task<Operation<BinaryData>> ImportPetAsync(RequestContent content, RequestOptions options = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("PetStoreClient.ImportPet");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateImportPetRequest(content);
+                return await LowLevelOperationHelpers.ProcessMessageAsync(Pipeline, message, _clientDiagnostics, "PetStoreClient.ImportPet", OperationFinalStateVia.AzureAsyncOperation, options).ConfigureAwait(false);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+#pragma warning disable AZC0002
+        public virtual Operation<Pet> ImportPet(Pet inputPet)
+#pragma warning restore AZC0002
+        {
+            var content = RequestContent.Create(inputPet);
+            var operation = (LowLevelFuncOperation<BinaryData>)ImportPet(content, default);
+            return new LowLevelFuncOperation<Pet>(operation, r => (Pet)r);
```

Comment:
Since this is a "grow-up method," it would be preferable not to return a type that is prefixed with "LowLevel."  Is that a possibility?

Reply:
Can we use the same type but change the type name `LowLevelFuncOperation<T>` as if `T` is `BinaryData`, then it's for low level, but I guess it could also work if `T` is a high level model?

Reply:
I'm open to that.  As I think about it more, too, as long as this is not a public type, we can change how it is named at any time later on.  We'll be re-naming LLC to something else in the not-too-distant future, and so should probably do a more holistic renaming as part of that.  So, let's keep it as it is for now, and maybe create an issue to track it?